make/compiler/Makefile.pgi

```diff
@@ -51,6 +51,8 @@ GEN_DYNAMIC_FLAG = -Bdynamic
 LIB_STATIC_FLAG =
 LIB_DYNAMIC_FLAG = -shared
 
+# TODO: set target architecture for optimization
+
```

Comment:
Oh wait.. nevermind my example. For a second I was thinking we'd thrown both flags to the backend, but that's not true, in the above example we wouldn't add anything to the compile line.

Might still be worth adding the NO_IEEE_FLOAT_GEN_CFLAGS anyways?